index.html

```diff
@@ -392,9 +340,8 @@ <h2 class="contact-title">Contact me</h2>
         </div>
       </div>
     </section>
-    <script src="hamberger.js"></script>
-	<script src="popup.js"></script>
-  <script src="form.js"></script>
+    <script src="hanbergwe.js"></script>
```

Comment:
- [ ] I also noticed that once on medium screen sizes, both the desktop and the mobile menu appear, please look into this as your design is meant to follow the provided Figma design, and also this is not great for user experience. Thanks :wave: :seedling: 
- SCREENSHOT: 
![Selection_010](https://user-images.githubusercontent.com/107632780/232638670-e06eb3ad-eeab-4546-ade6-309ecacdd2cd.png)

Reply:
It should always appear like this if you are not using a good internet connection. but only need to refresh the page and it will disappear.